docs/devguide/docs/tests.md

```diff
@@ -43,12 +43,12 @@ will randomly store `45697038-...-ed7a4a20e014` or `f02392e8-...-8a9a1ae5c47a` i
 
 ## Built-in Functions
 
-Predator supports some generic functions out of the box and they can be used to generate random data
+Predator supports some generic functions out of the box and they can be used to generate random data.
```

Comment:
lets add that everything generated here is a string

Reply:
but it's a bug. I think we should solve it. No?

Reply:
i have no idea... but at least lets say the truth for now